velox/functions/prestosql/Bitwise.h

```diff
@@ -175,6 +175,15 @@ struct BitwiseLogicalShiftRightFunction {
     // Presto defines this only for bigint, thus we will define this only for
     // int64_t.
     if (bits == 64) {
+      if (number < 0) {
+        // >> operator may perform an arithmetic shift right for signed
+        // integers, depending on the compiler, which gives wrong result when
+        // the input is negative. To ensure a logical shift, we cast it to
+        // uint64_t.
+        uint64_t unsignedNumber = static_cast<uint64_t>(number) >> shift;
+        result = static_cast<int64_t>(unsignedNumber);
+        return true;
+      }
       result = number >> shift;
       return true;
```

Comment:
nit : looks like we always return true, so we can just change signature to void ..